templates/repo/home.tmpl

```diff
@@ -73,6 +73,11 @@
 						</a>
 					</div>
 				{{end}}
+				<div class="fitted item mx-0">
+					<a href="{{.BaseRepo.Link}}/find/{{.BranchNameSubURL}}" class="ui compact basic button">
+						{{.i18n.Tr "repo.find_file.go_to_file"}}
```

Comment:
That is one approach.
I think another one could think of would be to display a magnifying glass icon ( and perhaps a tooltip).

Reply:
This layout is just copied from other buttons alongside it.

Reply:
Yes, but unlike other buttons, this one has an icon that you typically expect for this action.
I think most people are familiar with searching by clicking on a magnifying glass button.

Reply:
Texts are more clear. GitHub also only puts texts for "Go to file".

In Gitea, all buttons in this line are text-only, no icon.